### ISSUE TITLE
Jester (player) unlocked incorrectly

### ISSUE THREAD

**Issue by mbodrov** [TerryCavanagh/diceydungeons.com#2597]
Version 1.12.2
I'm having my first look at v.1.12 and for me the Jester playable character was not unlocked after I beat Jester as the final boss the first time.
He says his "You can try to escape and I'll join you" lines but nothing else happens.
He got unlocked after I beat him as the final boss for the 2nd time.
Last version I played before was an old version like v.1.7 and I seem to remember it working correctly there.
Expected behavior: Jester should be unlocked as playable character once you have defeated him as the final boss.

**Comment by TerryCavanagh** [comment]
Wow! That's pretty serious!

Did you maybe quit to title at any point before the save file registered it as complete? During the cutscene, perhaps?

**Comment by mbodrov** [comment]
No, it was a regular run with no interruptions, and I didn't notice anything out of the ordinary. As a programmer myself, I understand how difficult it can be to track down these issues but unfortunately there was nothing that looked suspicious.

**Comment by mbodrov** [comment]
On second thought, there was something. He said he would join me when I escape and then... he spun the wheel. I believe the wheel spin part was not supposed to happen, instead he's supposed to say something like "meh I'm not doing this anymore." That's what happened on his 2nd defeat, when he actually got unlocked. So it went off the rails somewhere around there.

**Comment by TerryCavanagh** [comment]
Ah, thank you - that's a helpful clue!
